### DISCUSSION TITLE
2.0.1 version no longer syncs, last block 5 496 001

### DISCUSSION THREAD

**Discussion by Ard5777** [Chia-Network/chia-blockchain#18210] [Support]
the only option get new db v2.

**Comment by BrandtH22** [comment]
Hey @Ard5777 , the latest version of the chia client only support the v2 db. 
There are two methods to upgrade the db based on the space you have available:
https://docs.chia.net/faq/#how-do-i-upgrade-my-version-1-blockchain-database-to-version-2

As a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia)
Let us know if any questions arise during the upgrade process.
